DevAcademy/Components/Toolbar/FilterBar/ButtonWithKindInToolbar.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftUI
+
+struct ButtonWithKindInToolbar: View {
```

Comment:
Tady to pojmenovani je taky trosku nestastne, protoze to vypada, ze se jedna o Button, ale Button to vlastne neni.
Mozna bych to prejmenoval na `KindToolbarButton` a pridal si mezi properties jeste `var action: () -> Void`. A v `body` by mohlo byt uz rovnou cele to Button, kteremu by se pouze predavala `action` a nastavil ten Text v `label`.